model/event/event.go

```diff
@@ -56,9 +56,6 @@ var (
 
 const resourceTypeKey = "r_type"
 
-func (e *EventLogEntry) UnmarshalBSON(in []byte) error { return mgobson.Unmarshal(in, e) }
```

Comment:
When using the legacy library, a nil list is stored in the DB as a non-nil list. I'm not sure if this something we depend on somewhere? Removing this didn't break any tests or the local event log pages so.

Reply:
Just kidding 🤦‍♀️  apparently I broke everything